concerto-behaviors.html

```diff
@@ -181,13 +181,19 @@
             this.setAttribute(key.toLowerCase(), JSON.stringify(data[key]));
           } else {
             this.setAttribute(key.toLowerCase(), data[key]);
+            // TODO! @gabe please look at this and see why the prior line is not setting all the base properties
```

Comment:
@gbprz this does not need resolved before it can be merged-- just something to look into and demystify for me if you are able to get to it.  Thanks.